### PR TITLE
web: create devconnect invite code page

### PR DESCRIPTION
**Pull request by nalinbhardwaj**
https://github.com/daimo-eth/daimo/assets/6984346/78fdb1a5-bb8b-4050-bcdb-e82b9bf8457b

TODO:
- [ ] add brief copy about Daimo from homepage?
- [x] test on mobile
- [x] add regular waitlist notion form link?

### CODE REVIEW

File: apps/daimo-web/src/app/[...topLevelPage]/route.tsx [hunk 1]
```diff
@@ -3,10 +3,10 @@ const superSo = "https://daimo.super.site";
...
   const url = new URL(request.url);
 
-  if (url.pathname === "/waitlist") {
-    // redirect to waitlist noteform for now
-    return Response.redirect("https://noteforms.com/forms/daimo-uk2fe4", 301);
-  }
+  // if (url.pathname === "/waitlist") {
+  //   // redirect to waitlist noteform for now
+  //   return Response.redirect("https://noteforms.com/forms/daimo-uk2fe4", 301);
+  // }
```
Comment (dcposch) [inline apps/daimo-web/src/app/[...topLevelPage]/route.tsx:9]: Delete

File: apps/daimo-web/src/app/waitlist/page.tsx [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+import { assert } from "@daimo/common";
+import Image from "next/image";
+import { useCallback } from "react";
+import toast, { Toaster } from "react-hot-toast";
+import { useZupass, ZupassProvider } from "zukit";
+
+import { PrimaryOpenInAppButton } from "../../components/buttons";
+import { RpcHookProvider, rpcHook } from "../../utils/rpcHook";
+
+const domain = process.env.NEXT_PUBLIC_DOMAIN || process.env.DAIMO_DOMAIN;
```
Comment (dcposch) [inline apps/daimo-web/src/app/waitlist/page.tsx:11]: Can import from env?

File: packages/daimo-api/src/api/zupass.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  });
+  if (response.status !== 200) return undefined;
+  return (await response.json()) as ZupassUser;
+}
+
+interface ZupassUser {
+  uuid: string;
+  commitment: string;
+  salt: string;
+  email: string;
```
Comment (dcposch) [inline packages/daimo-api/src/api/zupass.ts:41]: Can import from our fork of zukit?

**Review by dcposch — APPROVED** [review]
LGTM so far, feel free to prune then ship

**Comment by nalinbhardwaj** [comment]
Added regular waitlist notion form link
image

**Pull request merged**